pkg/kamaji/types.ts

```diff
@@ -1,2 +1,2 @@
-export const CLASTIX_PRODUCT_NAME = "Multitenancy Management";
-export const CLASTIX_NAME = "kamaji";
+export const CLASTIX_DASHBOARD = 'dashboard';
+export const CLASTIX_PRODUCT_NAME = 'kamaji';
```

Comment:
@bsctl could have some remarks on this renaming, he'll be back next week.

Reply:
+1